tests/infrared/16/metrics-qdr-connectors.yaml.template

```diff
@@ -2,6 +2,8 @@
 tripleo_heat_templates:
     - /usr/share/openstack-tripleo-heat-templates/environments/metrics/collectd-write-qdr.yaml
     - /usr/share/openstack-tripleo-heat-templates/environments/metrics/qdr-edge-only.yaml
+    - /usr/share/openstack-tripleo-heat-templates/environments/enable-legacy-telemetry.yaml
```

Comment:
I guess this is here for the redis setup? This is a dependency for ceilometer setup currently?